src/plugin.scss

```diff
@@ -19,182 +34,206 @@ $main-color: #fff;
-  $thumbnail-font-size: 18px;
+// Vertical/default playlist orientation
+.vjs-playlist-vertical {
 
-  font-size: $base-font-size;
-  background-color: $background-color;
-  color: $main-color;
-  padding: 0;
+  img {
+    width: 100%;
+    min-height: 54px;
+  }
 
   .vjs-playlist-item {
+    margin-bottom: 5px;
   }
 
   .vjs-playlist-thumbnail {
+    display: block;
     width: 100%;
   }
 
-  .vjs-selected .vjs-playlist-thumbnail img {
-    opacity: .2;
+  .vjs-playlist-thumbnail-placeholder {
+    height: 100px;
   }
+}
 
-  .vjs-playlist-thumbnail .vjs-playlist-now-playing-text {
-    display: none;
-    position: absolute;
-    top: 0;
-    left: 0;
-    margin: .5rem .8rem;
-    padding-left: 2px;
-  }
+// Horizontal playlist orientation
+.vjs-playlist-horizontal {
 
-  .vjs-selected .vjs-playlist-thumbnail .vjs-playlist-now-playing-text {
-    display: block;
+  img {
+    min-width: 100px;
+    height: 100%;
   }
 
-  .vjs-selected {
-    background-color: $highlight-color;
+  .vjs-playlist-item-list {
+    height: 100%;
+    white-space: nowrap;
   }
 
-  .vjs-up-next-text {
-    display: none;
+  .vjs-playlist-item {
+    display: inline-block;
+    height: 100%;
+    margin-right: 5px;
   }
 
-  .vjs-up-next .vjs-up-next-text {
-    text-transform: uppercase;
+  .vjs-playlist-thumbnail {
     display: block;
-    padding: .1rem 2px;
-    font-size: .8em;
+    height: 100%;
   }
 
-  time {
-    font-size: 1em;
+  .vjs-playlist-thumbnail-placeholder {
+    height: 100%;
+    width: 180px;
   }
+}
 
-  .vjs-playlist-title-container {
-    position: absolute;
-    bottom: 0;
-    padding: .5rem .8rem;
-    text-shadow:  1px  1px 2px black,
-                 -1px  1px 2px black,
-                  1px -1px 2px black,
-                 -1px -1px 2px black;
-  }
+// Prevent interaction with the playlist menu while ads are playing
+// on browsers that don't support pointer-events (IE<11), prevent scrolling
+// past the ad overlay
+.vjs-playlist.vjs-ad-playing {
+  overflow: hidden;
+}
+
+// prevent clicks and scrolling from affecting the playlist during ads
+.vjs-playlist.vjs-ad-playing.vjs-csspointerevents {
+  pointer-events: none;
+  overflow: auto;
 
-  .vjs-selected .vjs-playlist-title-container {
-    text-shadow: none;
+  // Fix for IE11 pointer-events bug that allowed clicking playlist during ads
+  .vjs-playlist-ad-overlay {
+    pointer-events: auto;
   }
+}
+
+// darken the playlist menu display to indicate it's not interactive during ads
+.vjs-playlist.vjs-ad-playing .vjs-playlist-ad-overlay {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
+  // IE8 fallback
+  background-color: #1a1a1a;
+  -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
+
+  // modern browsers
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+// Parametric rules. These are specialized for touch and mouse-based devices
+@mixin playlist-base($base-font-size: 14px) {
```

Comment:
The biggest change is moving almost everything from this mixin because it meant that a huge amount of static CSS was repeated for each use of the mixin.